docusaurus.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   baseUrl: "/",
   onBrokenLinks: "throw",
   onBrokenMarkdownLinks: "warn",
-  favicon: "img/favicon.ico",
```

Comment:
Similar question around the favicon.

Reply:
There actually wasn't a favicon---that's why this needed to change.